extensions/positron-python/python_files/positron/positron_ipykernel/tests/test_ui.py

```diff
@@ -214,3 +215,16 @@ def test_bokeh_show_sends_events(shell: PositronShell, ui_comm: DummyComm) -> No
     assert params["title"] == ""
     assert params["is_plot"]
     assert params["height"] == 0
+
+
+@pytest.mark.skipif(sys.version_info < (3, 9), reason="requires Python 3.9 or higher")
```

Comment:
I verified everything works fine with python 3.8 on my machine but for some reason I could not get things to work on CI with lower versions of python, I think due to how mangled I had to make the system requirements due to panel.

Reply:
Note: we're likely to drop Python 3.8 in a few months as it rolls out of support.